docs/pages/blog/mui-material-v5-3.md

```diff
@@ -0,0 +1,238 @@
+
+- `ownerState`: `ComponentProps` interface, eg. `ButtonProps`, `ChipProps`, etc.
+- `theme`: `Theme` interface from `@mui/material/styles`.
+
+```tsx
+{
+  MuiChip: {
+    styleOverrides: {
+      // ownerState: ChipProps
+      // theme: Theme
+      root: ({ ownerState, theme }) => ({...}),
+    },
+  }
+}
+```
+
+If you extend the interface via module augmentation like this:
+
+```ts
+declare module '@mui/material/Button' {
+  interface ButtonPropsVariantOverrides {
+    dashed: true;
+  }
+}
+```
+
+you will be able to see those props in `ownerState.variant` 🎉. `theme` can be augmented as well.
+
+## Experimental `sx` function
+
+Initially, `sx` was designed to be a prop that enables you to inject styles with a shorthand notation to components created with the `styled` API:
+
+```jsx
+import { styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+
+const Label = styled('span')({
+  fontWeight: 'bold',
+  fontSize: '0.875rem',
+})
+
+<Box sx={{ display: 'flex' }}>
+  <Label sx={{ color: 'text.secondary' }}>Label</Label>
+</Box>;
+```
+
+> 💡 All MUI components are created with `styled` API, so they accept `sx` prop by default.
+
+`sx` helps developers write less code and be more productive once they are familiar with the API. With the callback support in `styleOverrides`, it is now possible to use `sx` like syntax at the global theme overrides.
+
+All you need is to use the [`experimental_sx`](/system/styled/#how-can-i-use-the-sx-syntax-with-the-styled-utility) function:
+
+```jsx
+import {
+  ThemeProvider,
+  createTheme,
+  experimental_sx as sx,
+} from '@mui/material/styles';
+
+<ThemeProvider
+  theme={createTheme({
+    components: {
+      MuiChip: {
+        styleOverrides: {
+          root: sx({
+            px: '12px', // shorthand for padding-left & right
+            py: '6px', // shorthand for padding-top & bottom
+            fontWeight: 500,
+            borderRadius: '8px',
+          }),
+          label: {
+            padding: 0,
+          },
+        },
+      },
+    },
+  })}
+>
+  ...your app
+</ThemeProvider>;
+```
+
+If you want to create a dynamic style based on props, you can return an array from the callback:
+
+```js
+{
+  root: ({ ownerState }) => [
+    sx({
+      px: '12px',
+      py: '6px',
+      fontWeight: 500,
+      borderRadius: '8px',
+    }),
+    ownerState.variant === 'outlined' && ownerState.color === 'default' &&
+      sx({
+        borderColor: 'text.secondary',
+      }),
+    ownerState.size === 'small' &&
+      sx({
+        fontSize: { xs: '0.875rem', sm: '0.75rem' },
+      })
+  ],
+  // ...other keys
+}
+```
+
+<hr />
+
+**That's it for today!** Happy styling 💅.
+
+I hope this small update makes your customization experience better than before. Don't forget to share this update with your friends and/or colleagues.
+
+To get more updates like this in the future, **subscribe to our newsletter** at the bottom of this page.
+
+## Read more
+
```

Comment:
Should we add a link to the whole MUI Core v5.3.0 release?

Reply:
Added the link at the top of the post.